src/logic/profiles.ts

```diff
@@ -1,15 +1,40 @@
-import { Entity, Avatar } from '@dcl/schemas'
+import { Profile, ProfileAvatarsItem } from 'dcl-catalyst-client/dist/client/specs/lambdas-client'
+import { normalizeAddress } from '../utils/address'
 
-export function getProfileAvatar(profile: Pick<Entity, 'metadata'>): Avatar {
-  const [avatar] = profile.metadata.avatars
+export function getProfileAvatarItem(profile: Pick<Profile, 'avatars'>): ProfileAvatarsItem {
+  const [avatar] = profile.avatars ?? []
 
   if (!avatar) throw new Error('Missing profile avatar')
 
   return avatar
 }
 
-export function getProfilePictureUrl(baseUrl: string, { id }: Pick<Entity, 'id'>): string {
-  if (!baseUrl) throw new Error('Missing baseUrl for profile picture')
+export function getProfileName(profile: Pick<Profile, 'avatars'>): string {
+  const { name } = getProfileAvatarItem(profile)
 
-  return `${baseUrl}/entities/${id}/face.png`
+  if (!name) throw new Error('Missing profile avatar name')
+
+  return name
+}
+
+export function getProfileUserId(profile: Pick<Profile, 'avatars'>): string {
+  const { userId } = getProfileAvatarItem(profile)
+
+  if (!userId) throw new Error('Missing profile avatar userId')
+
+  return normalizeAddress(userId)
+}
+
+export function getProfileHasClaimedName(profile: Pick<Profile, 'avatars'>): boolean {
+  const { hasClaimedName } = getProfileAvatarItem(profile)
+  return hasClaimedName ?? false
+}
+
+export function getProfilePictureUrl(profile: Pick<Profile, 'avatars'>): string {
+  const { avatar } = getProfileAvatarItem(profile)
+  const { face256 } = avatar?.snapshots ?? {}
+
+  if (!face256) throw new Error('Missing profile avatar picture url')
+
+  return face256
 }
```

Comment:
Not for this PR, but I think we are trusting too much in the service that generates profile images. Can we discuss having a default profile image?

Currently, it seems that if a user (either the sender or the receiver) is missing a proper profile image, that user can't use these workflows. I'd try to change this behavior to keep `social-service` operational with a default image profile even when the image wasn't generated. At some point, it will get generated, and it will be rendered on UI, but even if it is missing, you can still use these workflows.